### PR TITLE
ROC-3343: Update response receipt PDF

### PR DESCRIPTION
**Pull request by andrewwa-kainos**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ROC-3343

### Change description ###
Update response receipt PDF

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 1]
```diff
@@ -739,41 +742,59 @@
...
+<div class="section">
+  <div class="section-heading">
+    <strong>Mediation</strong>
+  </div>
+  <div class="section-body">
+    <table>
+      <tr>
+        <td><strong>Willing to try mediation</strong></td>
+        <td>
+          {% if mediation %}Yes{% else %}No{% endif %}
```
Comment (timja) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: this would be shorter? `{{ "Yes" if mediation else "No" }}`

Comment (andrewwa-kainos) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: Nice, I saw a few variations on this

Comment (andrewwa-kainos) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: Alas, disnae work
`Unexpected token of value "if" and type NAME, expected token of type PRINT_END`

Comment (timja) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: damn mustn't work in the java library :(

Comment (andrewwa-kainos) [reply src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: But pebble doesn't support this notation; I'll go with a ternary
`{% mediation ? "Yes" : "No" %}`

File: src/main/resources/citizen/templates/document/defendantResponseReceipt.html [hunk 2]
```diff
@@ -281,45 +284,45 @@
...
             <td>
               <p>{{ amount }}</p>
             </td>
           </tr>
         {% else %}
           <tr>
-            <td>How much money have you paid?</td>
+            <td>Amount they have paid</td>
             <td>
               <p>{{ amount }}</p>
```
Comment (sabahirfan) [inline src/main/resources/citizen/templates/document/defendantResponseReceipt.html]: here remove `<p>`

**Pull request merged**